packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -71,20 +71,21 @@ const mapOptionsToQueryParams = (
   return { ...params, ...transitiveParams, ...tracking }
 }
 
-const getBaseUrl = (formString: string, chat: boolean = false): URL => {
+const getBaseUrl = (formString: string, hostname: string, chat: boolean = false): URL => {
   const prefix = chat ? 'c' : 'to'
+
   return new URL(
     formString.startsWith('http://') || formString.startsWith('https://')
       ? formString
-      : `${FORM_BASE_URL}/${prefix}/${formString}`
+      : `https://${hostname}/${prefix}/${formString}`
```

Comment:
I think we should support hostnames with protocol as well, in case you want to use eg. `http://localhost`. WDYT?

Reply:
I was thinking that for the time being we could still continue with passing the complete URL via the form id (just for the testing purposes) and remove the part completely once we implement `yarn preview` with HTTPS in Jarvis to avoid having a code just for the testing. Alternatively, we could provide `baseUrl` and specify the whole part even with the protocol.

WDYT @mathio? cc @trapped @lussn @Coralie-TF

Reply:
This works for me 🚀